docs/README.md

```diff
@@ -7,6 +7,12 @@ To start the docs site in development mode, from the project root, run:
 ```sh
 yarn && yarn start
 ```
+If you do not have yarn installed, select your OS and follow the instructions on the [Yarn website](https://yarnpkg.com/lang/en/docs/install/#mac-stable).
+
+*Mac users: note that you must have version 10.10.3 or higher for the Yarn installation to work as of 2019*
+
+*Also note: DO NOT USE NPM, use Yarn to install dependencies and everything else*
```

Comment:
While I would personally use yarn over npm without hesitation, I don't think that we should take a bold side here.

```suggestion
```

Reply:
It would be nice to know the issues you had when using npm instead of yarn.

Reply:
@isaacblinder Will you comment on this with the issues you had with npm?

Reply:
I have overlooked one thing, this is only for our documentation, yarn is required!

Reply:
I would imagine the problem is that npm doesn't support workspaces: https://stackoverflow.com/questions/46947557/do-yarn-workspaces-work-with-npm-too.